### ISSUE TITLE
This isn't a robot project

### ISSUE THREAD

**Issue by ar1staeus** [FRCTeamPhoenix/2017-preseason-vision#1]
I don't have the repo cloned on my computer or immediate access to a computer that can do that but this isn't a C++ robot project, could Ian or someone remove the files that were added? I might just remake the repo so those commits aren't in the history, not sure

**Comment by BenNF** [comment]
Wont vision need robot code when you are tracking a target from the Rio?
My initial plan for this repo would be for all vision code in all
languages.

**Comment by ar1staeus** [comment]
Possibly, but I don't think it makes sense. Having two different programs in one repo seems kind of messy and disorganized to me. I'll probably just fix this myself though, just want to make sure that everyone knows what I do and why.

**Comment by IanCoolidge0** [comment]
Yeah, if this isn't what you want for the vision code feel free to change this repo or make a different one. I don't know if you'd want to make a separate repo for the non-robot code though, and keep this one for utilizing it on a robot project (turning the robot to track a target, or maybe if someone wanted to look into camera switching again).
